corehq/apps/products/fixtures.py

```diff
@@ -52,8 +53,19 @@ def get_products():
                 key=lambda product: product.code
             )
 
-        return simple_fixture_generator(
+        fixture_nodes = simple_fixture_generator(
             restore_user, self.id, "product", PRODUCT_FIELDS, get_products, restore_state.last_sync_log
         )
+        if not fixture_nodes:
+            return []
+
+        if restore_state.params.openrosa_version and restore_state.params.openrosa_version < '2.1':
+            # Don't include index schema when openrosa version is specified and below 2.1
```

Comment:
I initially wrote this as:
```python
if not restore_state or openrosa_version > '2.1':
    # schema
else:
    # no schema
```
(happy to switch back if that's clearer)
The difference between this use-case and async_restore is that here we use the newer version also when openrosa_version isn't provided (Clayton was particular on that point).

I do like the idea of sticking to the map of features to openrosa versions, though - cleaner than just sticking it in app logic.  Think that still make sense if it's a discontinuous version range?